### PR TITLE
Serviceoffloading/knox handling

### PR DESCRIPTION
**Pull request by Chikdolman**
Adds Mouse, Keyboard input for Knox enabled devices.

### CODE REVIEW

File: chrome/android/BUILD.gn [hunk 1]
```diff
@@ -72,7 +72,11 @@ chrome_jni_registration_header =
 chrome_jni_for_test_registration_header = "$root_build_dir/gen/chrome/browser/android/chrome_jni_for_test_registration.h"
 
 jinja_template("chrome_public_android_manifest") {
-  input = "java/AndroidManifest.xml"
+  if (enable_service_offloading_knox) {
+    input = "java/AndroidManifest_service_offloading_knox.xml"
```
Comment (Sunwoo-Nam) [inline chrome/android/BUILD.gn]: I think we can use AndroidManifest.xml for Knox API.

File: content/browser/android/content_startup_flags.cc [hunk 2]
```diff
@@ -87,6 +87,12 @@ void SetContentCommandLineFlags(bool single_process) {
   // Prevents the renderer process from being killed for WebRTCGameStreaming.
   parsed_command_line->AppendSwitch(
       service_manager::switches::kNoSandbox);
+
+  // For using Knox in the WebRTC Service, permission must be passed to the
+  // renderer process. Currently, the Knox permission only applies to the
+  // browser process and it cannot be propagated to the renderer process.
+  // Therefore we apply single process mode for Knox system input.
+  parsed_command_line->AppendSwitch(switches::kSingleProcess);
```
Comment (Sunwoo-Nam) [inline content/browser/android/content_startup_flags.cc]: The single process mode should be enabled if |enable_service_offloading_knox| is true.

File: third_party/blink/renderer/modules/modules_idl_files.gni [hunk 3]
```diff
@@ -528,13 +528,16 @@ if (!is_android) {
                                      ],
                                      "abspath")
 }
-if (((is_android && enable_service_offloading_knox) || is_win) && enable_service_offloading) {
-  modules_idl_files += get_path_info(["input_control/input_control.idl"], "abspath")
+
+if (is_win || is_android) {
+  if(enable_service_offloading){
```
Comment (Sunwoo-Nam) [inline third_party/blink/renderer/modules/modules_idl_files.gni]: Why do you change the condition?

**Review by Sunwoo-Nam — COMMENTED** [review]
Please remove the commit for conflict.

File: third_party/blink/renderer/modules/modules_idl_files.gni (hunk 3 above)
Comment (Chikdolman) [reply third_party/blink/renderer/modules/modules_idl_files.gni]: Rolled back

File: content/browser/android/content_startup_flags.cc (hunk 2 above)
Comment (Chikdolman) [reply content/browser/android/content_startup_flags.cc]: Thank you. Done

File: chrome/android/BUILD.gn (hunk 1 above)
Comment (Chikdolman) [reply chrome/android/BUILD.gn]: THank you. Done

**Pull request closed without merging**